rt-shell/scripts/scr_shell_scripts/scr_shell_scripts.gml

```diff
@@ -30,6 +30,10 @@ function sh_help() {
 	)
 }
 
+function sh_clear() {
+	obj_shell.clear();
+}
```

Comment:
The `scr_shell_scripts` is just a set of examples, so adding `sh_clear` here won't get packaged and shipped with the bundle. I've been debating whether to have built-in functions, and if we should just declare them inside `obj_shell` as `sh_whatever` or if they should be done differently somehow. 

What are your thoughts?

Reply:
Yeah I was thinking about that too. I think built in functions would be a good idea. I was thinking about how to accomplish user defined autocompletion for their own functions and we could do structs containing the data linked to each function or something like that. Then I was thinking we could have the user group these however they want so for example, they could have a development grouping and then they could have a player grouping. That way, the shell could be actually shipped with games and provide users with functionality, but keep them away from game breaking functionality meant only for development.

Native functions like "clear" and others could be a part of a "native" grouping. I have a naive approach to this working for my own game but it's hard coded and I'd like to flesh this out to be more dynamic. That would probably have to be it's own PR though.

Reply:
Sounds like it's out of the scope of this PR. We'll leave it as-is for now, although maybe we'll fix it to have `clear` be a built-in function before we ship a release with this functionality.